### PR TITLE
Data generation script

### PR DESCRIPTION
**Pull request by Tina-Luk**

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -37,8 +37,8 @@ class App extends React.Component {
 		$.get(`http://localhost:3003/${id}/images`, (data) => {
 			console.log(data);
 			this.setState({
-				mainImage: data[id][0],
-				imagesList: data[id],
+				mainImage: data['images'][0],
+				imagesList: data['images'],
```
Comment (mbesio) [inline client/src/App.jsx:41]: are you hardcoding this id here because there is no image service running?

Comment (mbesio) [reply client/src/App.jsx:41]: probably it doesn't matter as this is not part of the db script - lol

File: database/dataGenerationSDC.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	const imageCount = Math.floor(Math.random() * 4 + 1);
+	const allImages = [];
+	for (let j = 0; j < imageCount; j += 1) {
+		const randomUrl = Math.floor(Math.random() * urls.length);
+		allImages.push(urls[randomUrl]);
+	}
+	const obj = {};
+	obj['images'] = allImages;
+	obj['item'] = String(id);
+	return obj;
```
Comment (mbesio) [inline database/dataGenerationSDC.js:30]: maybe use dot notation? Or I believe this is what eslint would say?

**Review by mbesio — COMMENTED** [review]
Looks good to me!

File: client/src/App.jsx (hunk 1 above)
Comment (Tina-Luk) [reply client/src/App.jsx:41]: Oh yeah this is the front end code. I made some modifications to the response object, so all entries have the same key's. Previously, she had the response with different key value's . I'm hardcoding the word 'images' to get the correct response urls.

File: database/dataGenerationSDC.js (hunk 2 above)
Comment (Tina-Luk) [reply database/dataGenerationSDC.js:30]: oh yeah good catch i can update that

**Pull request merged**